### DISCUSSION TITLE
Remeshing Error. Segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by ap264** [neperfepx/neper#540] [Q&A] [answered]
I am trying to repair a mesh which was created using tri-element type and -dim 2 with -rcl 0.6. But during the resmeshing process the module gives me an error saying the segmetation failed and core is dumped. Code used to create the voronoi cube and mesh is given below

```
neper -T -n 79 -morpho "diameq:custom(mydistrib)" -morpho "aspration(1,1,1.75)" -o mytess
neper -M mytess.tess -dim 2 -elttype tri -rcl 0.6
neper -M mytess.tess,mytess.msh -rcl 0.4 -o newmesh
```

Error:
![Screenshot from 2022-08-18 20-18-01](https://user-images.githubusercontent.com/109927295/185516627-93c07dea-9ca0-48fa-b705-0bebc0a08194.png)

**Comment by rquey** [comment, marked as answer]
Thanks for the bug report. Fixed in version 4.4.2-34.

**Reply by ap264** [reply]
Happy to help sir. 

I have a question. Do I have to use the command "wget https://github.com/neperfepx/neper/archive/refs/heads/main.zip" in linux module to update the software?

**Reply by rquey** [reply]
Possibly, yes, https://neper.info/downloads.html#downloads then https://neper.info/doc/introduction.html#installing-neper.

**Reply by ap264** [reply]
Thank you for the reply.
